src/content/peerconnection/restart-ice/js/test.js

```diff
@@ -27,13 +27,21 @@ function getTransportAddresses(stats) {
   var localAddress;
   var remoteAddress;
   stats.forEach(function(report) {
-    if (report.googActiveConnection === 'true') {
+    if (report.googActiveConnection === 'true' ||
+        report.state === 'succeeded') {
```

Comment:
I think with the new stats the approach needs to be different. You select the connection, then get the [selectedCandidatePairId](https://w3c.github.io/webrtc-stats/#dom-rtctransportstats-selectedcandidatepairid)

Which gets very complicated however so i'm ok with doing this (which works) in the test.